otel/assets/dashboards/otel_host_metrics_dashboard.json

```
@@ -476,9 +489,10 @@
                                     "response_format": "timeseries",
                                     "queries": [
                                         {
-                                            "query": "avg:otel.system.cpu.time{$host} by {host,cpu,state}",
+                                            "query": "avg:otel.system.cpu.time{$host} by {state}",
```

```suggestion
                                            "query": "avg:otel.system.cpu.time{$host} by {cpu,state}",
```
I guess this is just a typo?